lib/v2/19lou/index.js

```diff
@@ -0,0 +1,79 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const timezone = require('@/utils/timezone');
+const { parseDate } = require('@/utils/parse-date');
+const iconv = require('iconv-lite');
+
+function SetCookie(b, h) {
+    const c = new Date();
+    const g = SetCookie.arguments;
+    const e = SetCookie.arguments.length;
+    const d = e > 2 ? g[2] : null;
+    const i = e > 3 ? g[3] : null;
+    const f = e > 4 ? g[4] : null;
+    const a = e > 5 ? g[5] : false;
+    if (d !== null && d >= 0) {
+        c.setTime(c.getTime() + d * 24 * 60 * 60 * 1000);
+    }
+    return b + '=' + encodeURI(h) + (d === null || d < 0 ? (d === -1 ? '; expires=-1' : '') : '; expires=' + c.toGMTString()) + (i === null ? '' : '; path=' + i) + (f === null ? '' : '; domain=' + f) + (a === true ? '; secure' : '');
+}
```

Comment:
```suggestion
function SetCookie(cookieName, cookieValue) {
    const expires = new Date();
    const g = SetCookie.arguments;
    const e = SetCookie.arguments.length;
    const day = e > 2 ? g[2] : null;
    const path = e > 3 ? g[3] : null;
    const domain = e > 4 ? g[4] : null;
    const secure = e > 5 ? g[5] : false;
    if (day !== null && day >= 0) {
        expires.setTime(expires.getTime() + day * 24 * 60 * 60 * 1000);
    }
    return (
        cookieName +
        '=' +
        encodeURI(cookieValue) +
        (day === null || day < 0 ? (day === -1 ? '; expires=-1' : '') : '; expires=' + expires.toGMTString()) +
        (path === null ? '' : '; path=' + path) +
        (domain === null ? '' : '; domain=' + domain) +
        (secure === true ? '; secure' : '')
    );
}
```
More readability?

---

```js
        /**
         * Set cookie
         * @ param
         * @ return
         */
        setCookie: function(cookieName, cookieValue, seconds, path, domain, secure) {
            var expires = null;
            if (seconds != -1) {
                expires = new Date();
                expires.setTime(expires.getTime() + seconds);
            }
            document.cookie = [
                escape(cookieName),
                '=',
                escape(cookieValue),
                (expires ? '; expires=' + expires.toGMTString() : ''),
                (path ? '; path=' + path : '/'),
                (domain ? '; domain=' + domain : ''),
                (secure ? '; secure' : '')
            ].join("");
        },
```
Their `setCookie` function also escape `cookieName`.

Reply:
The snippet came from the source code of the website and I just copied it 😅. Thanks for reviewing. Your practice is much better.

Reply:
The above code is taken from
https://js2.citysbs.com/0.8.10.76/forum/App.component.js
I think you looked at the minified one: 
https://js2.citysbs.com/0.8.10.76/forum/AM.min.js

Reply:
You caught me 😄.